R/utils.R

```diff
@@ -114,31 +112,33 @@ parse_journal = function(xml_source, outcome, hits, multiple = FALSE,
     # TODO: check whether xml_find_first returns the policy with the highest
     # priority.
 
-    title = xml_text(xml_find_first(xml_source, "//jtitle"))
-    issn = xml_text(xml_find_first(xml_source, "//issn"))
+    title <- xml_text(xml_find_first(xml_source, "//jtitle"))
+    issn <- xml_text(xml_find_first(xml_source, "//issn"))
 
-    romeocolour = xml_text(xml_find_first(xml_source, "//romeocolour"))
-    romeocolour = ifelse(romeocolour == "gray", NA_character_, romeocolour)
+    romeocolour <- xml_text(xml_find_first(xml_source, "//romeocolour"))
+    romeocolour <- ifelse(romeocolour == "gray", NA_character_, romeocolour)
 
-    preprint = xml_text(xml_find_first(xml_source, "//prearchiving"))
-    preprint = ifelse(preprint == "unknown", NA_character_, preprint)
+    preprint <- xml_text(xml_find_first(xml_source, "//prearchiving"))
+    preprint <- ifelse(preprint == "unknown", NA_character_, preprint)
 
-    postprint = xml_text(xml_find_first(xml_source, "//postarchiving"))
-    postprint = ifelse(postprint == "unknown", NA_character_, postprint)
+    postprint <- xml_text(xml_find_first(xml_source, "//postarchiving"))
+    postprint <- ifelse(postprint == "unknown", NA_character_, postprint)
 
-    pdf = xml_text(xml_find_first(xml_source, "//pdfarchiving"))
-    pdf = ifelse(pdf == "unknown", NA_character_, pdf)
+    pdf <- xml_text(xml_find_first(xml_source, "//pdfarchiving"))
+    pdf <- ifelse(pdf == "unknown", NA_character_, pdf)
 
-    pre_embargo = parse_embargo(xml_source, "pre")
-    post_embargo = parse_embargo(xml_source, "post")
-    pdf_embargo = parse_embargo(xml_source, "pdf")
+    pre_embargo <- parse_embargo(xml_source, "pre")
+    post_embargo <- parse_embargo(xml_source, "post")
+    pdf_embargo <- parse_embargo(xml_source, "pdf")
 
     return(data.frame(title, issn, romeocolour,
                       preprint,    postprint,    pdf,
                       pre_embargo, post_embargo, pdf_embargo,
                       stringsAsFactors = FALSE))
 
-  } else if (outcome %in% c("manyJournals", "excessJournals")) {
+  } else if (outcome %in% c("manyJournals", "excessJournals") |
```

Comment:
Isn't this an `else` ? I'd say all cases are listed between L98 and this. You might want to explicitly keep this for clarity though. Your choice.